### PR TITLE
Minor update to README.md, updated imports and improved type casting

### PR DESCRIPTION
**Pull request by prm-darrow**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +79,7 @@ const apiClient = <Req, Res>(
 ) => async (request: Req): Promise<any> => {
   // AbortController was added in node v14.17.0 globally.
```
Comment (prm-dan) [inline README.md:80]: Please delete these comments.

File: README.md [hunk 2]
```diff
@@ -78,9 +79,7 @@ const apiClient = <Req, Res>(
...
   // This can brought in as a normal import too.
-  const AbortController = globalThis.AbortController || await import('abort-controller')
-
-  const controller = new AbortController();
+  const controller = new abort.AbortController();
```
Comment (prm-dan) [inline README.md:82]: AbortController is now in LTS.

Does the following work (if you remove the import statement)?
```
const controller = new AbortController();
```

Comment (prm-darrow) [reply README.md:82]: I think yes if the user has the `@types/node` package, which they should, no if they do not. If we remove the `abort.` then it makes the import line above removable.

Comment (prm-dan) [reply README.md:82]: Sounds good.  Please remove `abort.`

**Comment by prm-dan** [comment]
I'm not sure the latest.  I'll close out.

**Review by prm-dan — APPROVED** [review]

**Pull request merged**